legacy/copy_number.Rmd

```diff
@@ -0,0 +1,876 @@
+---
```

Comment:
I think this file/folder is legacy that has been dropped from the master. Would be good to do a diff against master and make sure no files were dropped